setup.cfg

```diff
@@ -26,13 +26,13 @@ extend-ignore = E203
 
 [isort]
 profile = black
-src_paths = ["src"]
+src_paths = ["src"]  # TODO: isort actually does nothing. When enabling it again, let's transition to ruff
```

Comment:
Setting that motivated the PR

Reply:
The underlying issue is not addressed here though, as almost every file would have a diff. Waiting for a time where we wouldn't cause merge conflicts.